lang/c++/impl/avrogencpp.cc

```diff
@@ -728,28 +719,15 @@ void CodeGen::generate(const ValidSchema &schema) {
     os_ << "#ifndef " << h << "\n";
     os_ << "#define " << h << "\n\n\n";
 
-    os_ << "#include <sstream>\n";
-#if __cplusplus >= 201703L
-    os_ << "#include <any>\n";
-#else
-    if (useCpp17_)
-        os_ << "#include <any>\n";
-    else
-        os_ << "#include \"boost/any.hpp\"\n";
-#endif
-    os_ << "#include \"" << includePrefix_ << "Specific.hh\"\n"
+    os_ << "#include <sstream>\n"
+        << "#include <any>\n"
+        << "#include \"" << includePrefix_ << "Specific.hh\"\n"
         << "#include \"" << includePrefix_ << "Encoder.hh\"\n"
         << "#include \"" << includePrefix_ << "Decoder.hh\"\n"
         << "\n";
 
-    vector<string> nsVector;
     if (!ns_.empty()) {
-        boost::algorithm::split_regex(nsVector, ns_, boost::regex("::"));
-        for (vector<string>::const_iterator it =
-                 nsVector.begin();
-             it != nsVector.end(); ++it) {
-            os_ << "namespace " << *it << " {\n";
-        }
+        os_ << "namespace " << ns_ << " {\n";
```

Comment:
Does this still work the same way ?
`ns_` is a String that was split by `::` in the old version of the code. Now it is just printed as is.

Reply:
C++17 [supports nested namespaces](https://stackoverflow.com/questions/11358425/is-there-a-better-way-to-express-nested-namespaces-in-c-within-the-header#28022457), so this should be fine to print as is.